src/integrationTest/java/net/sf/jabref/gui/ParameterizedMenuNewEntryTest.java

```diff
@@ -3,7 +3,7 @@
 import java.util.Arrays;
 import java.util.Collection;
 
-import net.sf.jabref.model.entry.EntryUtil;
+import net.sf.jabref.model.strings.StringUtil;
```

Comment:
Yes we now have a ModelStringUtil class, which already moved some string-related methods to model. I would propose to merge everything together: logic.StringUtil, model.ModelStringUtil and model.EntryUtil, they all deal with the same thing. I will create a separate PR with this change (because it makes merging easier).